community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/slurmsync.py

```diff
@@ -27,7 +27,7 @@
 import yaml
 import datetime as dt
 from datetime import datetime
-from typing import Dict, Tuple
+from typing import Dict, Tuple, Set
```

Comment:
unused?

Reply:
removed. I think we need to catch these things in pre-commits.